cosim/include/verilator/bp_zynq_pl.h

```diff
@@ -62,10 +62,10 @@ class bp_zynq_pl {
   //   the first eval, but BSG designs tend to do assertions on negedge
   //   at the least.
   static void tick(void) {
-    bsg_timekeeper::next();
     tb->eval();
     bsg_timekeeper::next();
     tb->eval();
+    bsg_timekeeper::next();
```

Comment:
some kind of comment explaining why this is more correct

Reply:
yeah this is interesting -- I haven't fully debugged the cause. But, it seems that Verilator has ill-defined semantics for generated clocks in comparison with other DPI calls (https://github.com/verilator/verilator/issues/3278). When I changed the AXIL read to happen after a cycle instead of immediately, this symptom showed up.

Probably the safest move is to do

```
tb->eval();
bsg_timekeeper::next()
tb->eval()
bsg_timekeeper::next()
tb->eval()
```

but that may slow the simulation by 1/3...